Sources/FoundationExtensions.swift

```diff
@@ -24,29 +26,33 @@ extension NotificationCenter {
 	///         will terminate immediately with an `interrupted` event.
 	///         Otherwise, the producer will not terminate naturally, so it must
 	///         be explicitly disposed to avoid leaks.
-	public func rac_notifications(forName name: Notification.Name?, object: AnyObject? = nil) -> SignalProducer<Notification, NoError> {
+	public func notifications(forName name: Notification.Name?, object: AnyObject? = nil) -> SignalProducer<Notification, NoError> {
 		// We're weakly capturing an optional reference here, which makes destructuring awkward.
 		let objectWasNil = (object == nil)
-		return SignalProducer { [weak object] observer, disposable in
+		return SignalProducer { [base, weak object] observer, disposable in
```

Comment:
Done in https://github.com/ReactiveCocoa/ReactiveSwift/pull/51/commits/046c65192655576c1cbd0ecb95d66ffe776f5d20.